src/main/java/net/sf/jabref/logic/autocompleter/ContentAutoCompleters.java

```diff
@@ -12,12 +12,11 @@
 
 public class ContentAutoCompleters extends AutoCompleters {
 
-    AutoCompletePreferences preferences;
     private final JournalAbbreviationLoader abbreviationLoader;
 
 
     public ContentAutoCompleters(AutoCompletePreferences preferences, JournalAbbreviationLoader abbreviationLoader) {
-        this.preferences = Objects.requireNonNull(preferences);
+        Objects.requireNonNull(preferences);
```

Comment:
OK! Now, I see the bigger picture of how it is used. But could indeed be remove here.